### PR TITLE
Ddpb 801 view add client contacts

### PR DESCRIPTION
**Pull request by slizzio**

### CODE REVIEW

File: src/AppBundle/Form/Pa/ClientContactType.php [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+        $resolver->setDefaults(
+            [
+                'validation_groups' => ['add_clientcontact'],
+                'translation_domain' => 'client-contacts',
+                'data-class' => EntityDir\ClientContact::class
+            ]
+        );
+    }
+
+//    /**
```
Comment (Biggsen) [inline src/AppBundle/Form/Pa/ClientContactType.php]: This commented out code should be deleted

File: src/AppBundle/Resources/translations/validators.en.yml [hunk 2]
```diff
@@ -42,11 +42,7 @@ user:
   address2:
     notBlank: Enter your address
   address3:
-    notBlank: Enter your address
-  addressPostcode:
-    notBlank: Enter your postcode
-    minLength: The postcode must be at least {{ limit }} letters long
-    maxLength: The postcode can't be longer than {{ limit }} letters
+    notBlank: Enter your addressfirsN
```
Comment (Biggsen) [inline src/AppBundle/Resources/translations/validators.en.yml]: This looks like a typo

File: src/AppBundle/Resources/translations/validators.en.yml [hunk 3]
```diff
@@ -180,6 +176,27 @@ document:
...
+  form:
+    firstname:
+        notBlank: Enter the contact's first name
+        minMessage: The first name must be at least {{ limit }} letters long
+        maxMessage: The first name can't be longer than {{ limit }} letters
+    lastname:
+        notBlank: Enter the contact's last name
+        minMessage: The last name must be at least {{ limit }} letters long
+        maxMessage: The last name can't be longer than {{ limit }} letters
+    phone:
```
Comment (Biggsen) [inline src/AppBundle/Resources/translations/validators.en.yml]: Don't think this Phone validator is required for Contact

File: src/AppBundle/Resources/translations/validators.en.yml [hunk 4]
```diff
@@ -180,6 +176,27 @@ document:
...
+        notBlank: Enter the contact's phone number
+        minMessage: The last name must be at least {{ limit }} letters long
+        maxMessage: The last name can't be longer than {{ limit }} letters
+    email:
```
Comment (Biggsen) [inline src/AppBundle/Resources/translations/validators.en.yml]: email: notBlank shouldn't be required here

File: src/AppBundle/Form/Pa/ClientContactType.php (hunk 1 above)
Comment (slizzio) [reply src/AppBundle/Form/Pa/ClientContactType.php]: removed

File: src/AppBundle/Resources/translations/validators.en.yml (hunk 2 above)
Comment (slizzio) [reply src/AppBundle/Resources/translations/validators.en.yml]: un-typoed

File: src/AppBundle/Resources/translations/validators.en.yml (hunk 3 above)
Comment (slizzio) [reply src/AppBundle/Resources/translations/validators.en.yml]: yup, will never be used. removed

File: src/AppBundle/Resources/translations/validators.en.yml (hunk 4 above)
Comment (slizzio) [reply src/AppBundle/Resources/translations/validators.en.yml]: ditto, removed

**Pull request merged**